src/unpack.js

```diff
@@ -64,6 +64,25 @@ function unpackRelease(unpackFolder, targetFolderName, archiveFile, progressElem
 
                 unpacker = path.join(remote.app.getAppPath(), "tools/7zip/7za");
             }
+        } else if(os.platform() === "darwin"){
+            // Find from PATH
+            unpacker = which.sync("7za", {nothrow: true});
+
+            if(!unpacker){
+                // Some users on Mac import Linux versions of 7zip through homebrew //
+                console.log("No 7za found in PATH, using packed in one");
+
+                // Popular 7zip alternative for Mac users, linked on 7zip page at line 81 //
+                unpacker = path.join(remote.app.getAppPath(),
+                    "tools/7zip/Keka.app/Contents/MacOS/keka7z");
```

Comment:
You refer to this, but you didn't include the actual executable?
Why not use the version from homebrew? At work I've found homebrew to be one of the things that  makes using mac tolerable.